### PR TITLE
Added Chapeau 2016 / Lynton Wolfe 2016 / Devra 2016 / undo version

### PR DESCRIPTION
**Pull request by Witaminer**
As discused, undo -2016 numbering, simply -1 -2 -3 numbering for character badges, undo version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-ingress",
   "description": "Ingress commands for Hubot",
-  "version": "0.19.0",
+  "version": "0.19.1",
```
Comment (therealklanni) [inline package.json]: Please revert this change.

**Comment by EvoldicA** [comment]
@Witaminer unfortunately due to adding contributing rules your branch is now out of sync with the master.  Can you please merge the changes from the master?

**Comment by therealklanni** [comment]
@Witaminer I'm going to merge your changes from my own branch where I squashed your commits. Thanks very much for your contribution!
